Localize/localize-pipeline.yml

```diff
@@ -2,20 +2,114 @@ name: $(Date:MMddyy)$(Rev:.rrrr)
 
 trigger: none
 
+schedules:
+- cron: 0 8 * * Mon # mm HH DD MM DW
+  displayName: Localization update
+  branches:
+    include: 
+    - Localization
+  always: true
+
 stages:
 - stage: __default
   jobs:
-  - job: Job1
+  - job: localizationUpdate
+    displayName: 'Update localization'
     pool:
       vmImage: windows-latest
     steps:
+
+    - checkout: self
+      persistCredentials: true
+
+    - powershell: |
+        npm install
+      displayName: 'npm install'
+      
+    - powershell: |
+        $sprintInfo = Invoke-WebRequest https://whatsprintis.it -Headers @{"Accept"= "application/json"} | ConvertFrom-Json
+        if (($env:PR_CREATION_ENABLED -eq 'True') -and (($sprintInfo.week -eq 3) -or ($env:BUILD_REASON -eq 'Manual')))
+        {
+          Write-Host "shouldCreatePR was set to true"
+          Write-Host "##vso[task.setvariable variable=shouldCreatePR]$($true)"  
+        }
+        else
+        {
+          Write-Host "shouldCreatePR was set to false"
+          Write-Host "##vso[task.setvariable variable=shouldCreatePR]$($false)"
+        }
+        Write-Host "##vso[task.setvariable variable=sprint]$($sprintInfo.sprint)"
+      displayName: "Determine the number of the week in the sprint and sprint number"
+
+    - powershell: |
+        git config --global user.email "$(github_email)"
+        git config --global user.name "$(username)"
```

Comment:
exit codes for each git command should be checked and stop if non-zero

Reply:
Since the current task doesn't include the rework of PS scripts that we use here and in another repos, I'll resolve this, but we'll consider these changes in the future rework. Thanks!